### ISSUE TITLE
[Suggestion]:  Beds explode when trying to use them in the Bowels.

### ISSUE THREAD

**Issue by ghost** [nonamecrackers2/crackers-wither-storm-mod#1817]
### Intentional Mod Design

Currently interacting with a bed in the Bowels does nothing.

Beds explode when you try to use them in the Nether and the End. So likewise, beds should also explode when trying to use them in the Bowels.

**Comment by TaintedBlock** [comment]
You're still in the overworld though so it wouldn't make much sense

**Comment by Bloxxify2827** [comment]
Nooooo I want to live in the bowels

**Comment by RepositoryLover** [comment]
> Nooooo I want to live in the bowels

That's not the point of the bowels. That's not it's purpose.

**Comment by ghost** [comment]
> That's not the point of the bowels. That's not it's purpose.

But it's funny to live there.

**Comment by RepositoryLover** [comment]
> > That's not the point of the bowels. That's not it's purpose.
> 
> But it's funny to live there.

I guess.

**Comment by FireCode125** [comment]
You’re still technically in the overworld though

**Comment by Herobrine982950291** [comment]
Yes you are in the OverWorld, you just enter the butt of a creature.

**Comment by BluePineappleGuy** [comment]
> Yes you are in the OverWorld, you just enter the butt of a creature.

bro did not have to put it like that

**Comment by Steve982950291** [comment]
alright alright. but you get the idea... right?

**Comment by Herobrine982950291** [comment]
Alright alright... but you get what i mean, right?

**Comment by BladeLightsteel** [comment]
imagine riding inside the wither storm like a Trojan horse as like a moveable raider base

**Comment by Herobrine982950291** [comment]
> imagine riding inside the wither storm like a Trojan horse as like a moveable raider base

what does that have to do with this?

**Comment by BladeLightsteel** [comment]
nothing its just kinda funny

**Comment by Aimyx** [comment]
y'know, it might just be an "[Intentional Game Design]"

**Comment by Ch1216** [comment]
Something should happen to indicate that beds dont work in the bowels
